mobify/astro-scaffold
# Upgrading to android studio 2.0

Upgrade to Android studio 2.0

JIRA: None
Linked PRs: https://github.com/mobify/astro/pull/459

## Changes
- Update gradle version, android gradle plugin version and remove any mention of coveralls

## How to test-drive this PR
- `npm link astro-sdk` (develop version of `astro-sdk`) ... let me know if you have issues with this
- run the app and ensure it builds and runs

## TODOS:
